### PR TITLE
chore: renderer test coverage

### PR DESCRIPTION
**Pull request by muselesscreator**
Test coverage updates.  Starting with renderer components and moving down the coverage list.

### CODE REVIEW

File: src/containers/ReviewModal/ReviewErrors/LockErrors.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import ReviewError from './ReviewError';
  */
 export class LockErrors extends React.Component {
   get errorProp() {
-    if (this.errorStatus === ErrorStatuses.forbidden) {
+    if (this.props.errorStatus === ErrorStatuses.forbidden) {
```
Comment (leangseu-edx) [inline src/containers/ReviewModal/ReviewErrors/LockErrors.jsx:19]: 👍

File: src/utils/StrictDict.js [hunk 2]
```diff
@@ -8,10 +8,6 @@ const strictGet = (target, name) => {
     return target[name];
   }
 
-  if (name === '$$typeof') {
```
Comment (leangseu-edx) [inline src/utils/StrictDict.js:11]: How about `$$typeof = Symbol.for('StrictDict')`? Not that it is critical.

**Review by leangseu-edx — APPROVED** [review]
LGTM

File: src/utils/StrictDict.js (hunk 2 above)
Comment (muselesscreator) [reply src/utils/StrictDict.js:11]: actually, looks like this is just not necessary anymore.  typeof just works.  :shrug:

**Pull request merged**